Doc/library/sqlite3.rst

```diff
@@ -515,6 +515,16 @@ Connection Objects
 
       .. XXX what's a db_row-based solution?
 
+   .. method:: statement_cache()
+
+      Returns the LRU statement cache for this connection.  The statement
```

Comment:
What's the reason for adding a public API? Previously, statement caching was an implementation detail apart from `cached_statements` of `sqlite3.connect()` (which IMO was a bad idea)

Reply:
I used it for debugging and felt it was a nice feature. Looking at it now, I see no reason to add this to the API. I'll remove it. As you say, caching is an implementation detail.

Reply:
Fixed in 55342a89168a48bbc14f0a314c801509e453eff1. I see no way to explicitly test the caching now, so I removed the tests.